tfhe/src/integer/ciphertext/compact_list.rs

```diff
@@ -432,6 +436,10 @@ impl CompactCiphertextList {
         ))
     }
 
+    pub fn expand_with_casting(&self) {
+        todo!()
+    }
+
```

Comment:
Just to be sure, this is a todo for the next PR ?

Reply:
I can remove it from here, at first I thought of doing everything at once but it seemed too much, so yes will be done in the next PR, I'll remove this to keep a consistent code base